src/SonarScanner.MSBuild.PostProcessor/MSBuildPostProcessor.cs

```diff
@@ -218,6 +206,8 @@ private bool CheckCredentialsInCommandLineArgs(AnalysisConfig config, IAnalysisP
             return true;
         }
 
+#if NET46
```

Comment:
I assume this is an unrelated change you decide to make at the same time.

Could you make life easier for the reviewers by specifically calling miscellaneous changes out in review comments please?

Reply:
Yes, it was unrelated clean as you code. Those two methods appeared as unused under the default `netstandard2.0` VS view.
I'm sorry for making it more difficult, I tried to highlight the core changes in the PR description.

Reply:
The high-level description was useful; inline review comments would also help.